packages/ui/primitives/signature-pad/signature-pad.tsx

```diff
@@ -210,7 +210,33 @@ export const SignaturePad = ({
 
       onChange?.($el.current.toDataURL());
     }
-  };
+  }, [lines, defaultValue, onChange, perfectFreehandOptions]);
+
+  const handleKeyDown = useCallback(
```

Comment:
We should move the keyDown method into the useEffect callback so we don't need to use `useCallback` doing this is going to cause a lot of registrations and de-registrations as the arguments changes.